pkg/apis/toolchain/v1alpha1/notification_types.go

```diff
@@ -16,6 +16,8 @@ const (
 	// Status condition reasons
 	NotificationSentReason          = "Sent"
 	NotificationDeletionErrorReason = "UnableToDeleteNotification"
+	NotificationContextErrorReason  = "NotificationContextError"
+	NotificationDeliveryErrorReason = "NotificationDeliveryError"
```

Comment:
I'm curious - do we really need to have the word "notification" as part of the reason? It is the status of the Notification CR, I'd say that it's logical that the errors relate to the notification
```suggestion
	NotificationContextErrorReason  = "ContextError"
	NotificationDeliveryErrorReason = "DeliveryError"
```

Reply:
"NotificationContext" is an actual type, so for this one it's necessary.  For the delivery error though it's not required - I actually like "DeliveryError" as an alternative, so I'll change it.  Thanks :)